<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>estoque</class>
 <widget class="QMainWindow" name="estoque">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1024</width>
    <height>768</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Estoque</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="5" column="1">
     <layout class="QVBoxLayout" name="verticalLayout_6">
      <item>
       <widget class="QSpinBox" name="spinBox_2">
        <property name="styleSheet">
         <string notr="true">font: 10pt &quot;Georgia&quot;;</string>
        </property>
        <property name="maximum">
         <number>1000</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="styleSheet">
         <string notr="true">font: 10pt &quot;Georgia&quot;;</string>
        </property>
        <property name="text">
         <string>Atualizar quantidade</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="3" column="0" rowspan="3">
     <widget class="QTableView" name="estoque_table">
      <property name="styleSheet">
       <string notr="true">font: 10pt &quot;Georgia&quot;;</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QPushButton" name="b_voltar">
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 10pt &quot;Georgia&quot;;</string>
      </property>
      <property name="text">
       <string>Voltar</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLineEdit" name="lineEdit">
          <property name="styleSheet">
           <string notr="true">font: 10pt &quot;Georgia&quot;;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinBox">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>78</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 10pt &quot;Georgia&quot;;</string>
          </property>
          <property name="maximum">
           <number>1000</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QPushButton" name="b_adicionar">
          <property name="styleSheet">
           <string notr="true">font: 10pt &quot;Georgia&quot;;</string>
          </property>
          <property name="text">
           <string>Adicionar Item</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="b_remover">
          <property name="styleSheet">
           <string notr="true">font: 10pt &quot;Georgia&quot;;</string>
          </property>
          <property name="text">
           <string>Remover Item</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="4" column="1">
     <layout class="QVBoxLayout" name="verticalLayout"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>b_voltar</tabstop>
  <tabstop>lineEdit</tabstop>
  <tabstop>spinBox</tabstop>
  <tabstop>b_adicionar</tabstop>
  <tabstop>b_remover</tabstop>
  <tabstop>spinBox_2</tabstop>
  <tabstop>pushButton</tabstop>
  <tabstop>estoque_table</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>b_voltar</sender>
   <signal>clicked()</signal>
   <receiver>estoque</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>79</x>
     <y>42</y>
    </hint>
    <hint type="destinationlabel">
     <x>425</x>
     <y>124</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
